app/layout/aside_feed.phtml

```diff
@@ -63,11 +63,12 @@
 		<?php
 			foreach ($this->categories as $cat) {
 				$feeds = $cat->feeds();
+				$priority = $cat->attributes('priority');
 				if (!empty($feeds)) {
 					$c_active = FreshRSS_Context::isCurrentGet('c_' . $cat->id());
 					$c_show = $c_active || FreshRSS_Context::$user_conf->display_categories;
 		?>
-		<li class="tree-folder category<?= $c_active ? ' active' : '' ?>" data-unread="<?= $cat->nbNotRead() ?>">
+		<li class="tree-folder category<?= $c_active ? ' active' : '' ?><?= null === $priority ? '' : " priority_$priority"?>" data-unread="<?= $cat->nbNotRead() ?>">
```

Comment:
I have not checked in details, but I suggest using `data-priority="..."`

Reply:
The idea here is to be able to target them with a CSS rule. But I can also add some data.

Reply:
@Frenzie I am not sure I follow you thinking. Could you explain me what you have in mind?

Reply:
Sorry, I figured you knew. That's a CSS attribute selector.

https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

It's been supported by all browsers for close to 15 years.

Reply:
@Frenzie I knew but I didn't understand what you meant. Now I do :)

Reply:
We are already using CSS rules targetting data properties, e.g. https://github.com/FreshRSS/FreshRSS/blob/dev/p/themes/base-theme/template.css#L1023